contracts/amm/pool/Joseph.sol

```diff
@@ -48,21 +55,24 @@ abstract contract Joseph is JosephInternal, IJoseph {
         }
     }
 
-    function _checkVaultReservesRatio() internal view returns (uint256) {
-        (uint256 totalBalance, uint256 wadMiltonAssetBalance) = _getIporTotalBalance();
-        require(totalBalance > 0, JosephErrors.STANLEY_BALANCE_IS_EMPTY);
-        return IporMath.division(wadMiltonAssetBalance * Constants.D18, totalBalance);
-    }
-
     function _provideLiquidity(
         uint256 assetAmount,
         uint256 assetDecimals,
```

Comment:
You can get rid of `assetDecimals `. You always use `_getDecimals()`

Reply:
done